fe/fe-core/src/main/java/org/apache/doris/nereids/rules/exploration/OrExpansion.java

```diff
@@ -72,7 +72,7 @@ public class OrExpansion extends OneExplorationRuleFactory {
 
     @Override
     public Rule build() {
-        return logicalJoin().when(JoinUtils::shouldNestedLoopJoin)
+        return logicalJoin(any(), any()).when(JoinUtils::shouldNestedLoopJoin)
```

Comment:
better to move this file from exploration to rewriter dir?

Reply:
done